modules/installation/pages/server-setup.adoc

```diff
@@ -9,16 +9,17 @@ This section covers {productname} Server setup, using these procedures:
 * Synchronize the {sle} product channel from {scc}
 
 [WARNING]
-.Third Party Software
+.Installation of software on {productname} hosts
 --
 {productname} is part of the {sle} 4.0 product family and thus compatible with the software shipped with {sles}.
 
-{productname} is a complex system, and therefore installing third party software is not allowed.
+{productname} is a complex system, and therefore installing software is not allowed.
```

Comment:
> I don't think you can say that you can't install _any_ software, since that's primarily what suma is used for.

SUSE Manager is used for installing software _on other client systems_, not on the Server or the Proxy.

Reply:
I think this needs to be much more explicit, then.